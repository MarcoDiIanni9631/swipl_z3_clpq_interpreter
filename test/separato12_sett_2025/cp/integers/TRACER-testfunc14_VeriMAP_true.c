# 1 "MAP/SAFE-exbench/TRACER-testfunc14.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/TRACER-testfunc14.tmp.c"
# 23 "MAP/SAFE-exbench/TRACER-testfunc14.tmp.c"
//extern int unknown();

void bar(){
  int i,NONDET,q,z;
  i=0;

  if (q>0) z=4;
  else z=5;

  while (NONDET){
    i++;
  }
  return;
}

main(){
  int p,x;

  if (p>0) x=1;
  else x=3;

  bar();

  __VERIFIER_assert(!( x==2 ));

}
