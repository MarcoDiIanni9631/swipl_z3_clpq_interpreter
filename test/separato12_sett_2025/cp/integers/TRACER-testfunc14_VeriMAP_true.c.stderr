MAP/SAFE-exbench/TRACER-testfunc14.tmp.c:46: Warning: Body of function main falls-through. Adding a return statement
